packages/myjwt/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=myjwt
+_pkgname=MyJWT
+pkgver=119.f26dd19
+pkgrel=1
+pkgdesc='This cli is for pentesters, CTF players, or dev. You can modify your jwt, sign, inject, etc...'
```

Comment:
Use only one dot at the end of the phrase.

Reply:
Fixed.

Reply:
Do you forget to commit?

Reply:
No commented before commit; sorry about the confusion there ;)